components/authn-service/tokens/pg/pg_test.go

```diff
@@ -479,6 +479,69 @@ func TestCreateToken(t *testing.T) {
 	}
 }
 
+func TestPurgeProject(t *testing.T) {
+	ctx := context.Background()
+	store, db := setup(t)
+	assert := assert.New(t)
+
+	cases := map[string]func(*testing.T){
+		"when there are mulitple tokens, some with, some without the project to purge": func(t *testing.T) {
```

Comment:
It'd be easier to read if we split this test a few smaller tests, with the expected behavior in the tests name